lib/ClangImporter/ClangImporter.cpp

```diff
@@ -3033,53 +3033,22 @@ ClangNode ClangImporter::getEffectiveClangNode(const Decl *decl) const {
 void ClangImporter::lookupTypeDecl(
     StringRef rawName, ClangTypeKind kind,
```

Comment:
I should either remove `ClangTypeKind` from the API for factor it in to the lookup if it's necessary.